streaming/index.js

```diff
@@ -150,23 +132,24 @@ const redisConfigFromEnv = (env) => {
   const redisNamespace = env.REDIS_NAMESPACE || null;
 
   const redisParams = {
-    socket: {
-      host: env.REDIS_HOST || '127.0.0.1',
-      port: env.REDIS_PORT || 6379,
-    },
-    database: env.REDIS_DB || 0,
-    password: env.REDIS_PASSWORD || undefined,
+    host: env.REDIS_HOST || '127.0.0.1',
+    port: env.REDIS_PORT || 6379,
+    options: {
+      db: env.REDIS_DB || 0,
+      password: env.REDIS_PASSWORD || undefined,
+    }
   };
 
-  if (redisNamespace) {
-    redisParams.namespace = redisNamespace;
+  if (env.REDIS_URL && env.REDIS_URL.startsWith('unix://')) {
```

Comment:
This does still need a line of documentation "Setting redisParams.path overrides using host & port"